CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 
 ## [Unreleased]
+### Fixed
+- Pie Chart - Fix groupBy/aggregate on a field with custom column name.
+- Time Chart - Fix groupBy/aggregate on a field with custom column name.
+- Leaderboard Chart - Fix groupBy on a field with custom column name.
+- List Records - Fix sort on a field with custom column name.
```

Comment:
I am not convinced that this is the exact fix description as the sorting on a "custom column name" already works properly.
Do you talk about the sorting on belongsTo fields having a "reference field" with a "custom column name"?

Reply:
Yes it does fix it. I added some comment to the PR ;)